src/api/index.ts

```diff
@@ -585,6 +585,70 @@ claimSellerFundsFromSeveralAuctions.sendTransaction = async (
   return DutchX.claimTokensFromSeveralAuctionsAsSeller.sendTransaction(sellArr, buyArr, finalIndices, userAccount)
 }
 
+export const claimAndWithdrawSellerFundsFromSeveralAuctions: ClaimSellerFundsFromSeveralAuctions = async (
+  sell: DefaultTokenObject,
+  buy: DefaultTokenObject,
+  userAccount?: Account,
+  indices: number = 0,
+) => {
+  const { DutchX } = await dxAPI()
+  userAccount = await fillDefaultAccount(userAccount)
+
+  const claimableIndices = (
+    await DutchX.getIndicesWithClaimableTokensForSellers({ sell, buy }, userAccount, indices)
+  )[0].map(i => i.toNumber())
```

Comment:
I wonder if it would be better to have `.toString()` here, because some years in the future indexes will overflow?